setup.py

```diff
@@ -33,6 +33,9 @@
     "flake8",
     "types-requests",
     "mypy",
+    # for check of different parsers in recorder
+    "tomli; python_version < '3.11'",
+    "tomli-w",
```

Comment:
this is only test dependency, I just added it because `toml` looks to be more tricky format (eg parsing None values) and someone already might have started using it.